engine/src/main/java/org/terasology/rendering/assets/shader/ShaderProgramFeature.java

```diff
@@ -21,18 +21,12 @@
 /**
  */
 public enum ShaderProgramFeature {
-    FEATURE_REFRACTIVE_PASS
-            (0b00000001),
-    FEATURE_ALPHA_REJECT
-            (0b00000010),
-    FEATURE_LIGHT_POINT
-            (0b00000100),
-    FEATURE_LIGHT_DIRECTIONAL
-            (0b00001000),
-    FEATURE_USE_MATRIX_STACK
-            (0b00100000),
-    FEATURE_USE_FORWARD_LIGHTING
-            (0b01000000);
+    FEATURE_REFRACTIVE_PASS (0b00000001),
+    FEATURE_ALPHA_REJECT (0b00000010),
+    FEATURE_LIGHT_POINT (0b00000100),
+    FEATURE_LIGHT_DIRECTIONAL (0b00001000),
+    FEATURE_USE_MATRIX_STACK (0b00100000),
+    FEATURE_USE_FORWARD_LIGHTING (0b01000000);
```

Comment:
I wonder if it would be horribly inappropriate here to line up (with spaces) the binary codes. @Cervator? I seem to remember we had a discussion about spaces/tabs once. What do you think about this case?